src/cache.py

```diff
@@ -31,7 +32,7 @@ def refresh(self):
                 self._cache[key_pair.client_side_key] = self.fetch_document(
                     key_pair.server_side_key
                 )
-            except requests.exceptions.HTTPError:
+            except (requests.exceptions.HTTPError, orjson.JSONDecodeError):
                 received_error = True
                 logger.exception(
                     f"Received non 200 response for {key_pair.client_side_key}"
```

Comment:
This message is slightly inaccurate now. Even if the server returns 200 but some invalid data that can't be parsed by `orjson` the log will say non 200 response. Can we either change the log message to be more generic or catch `orjson` error in a different except?

Reply:
Yes, good catch. I've used a more generic error message. Since we're using `logger.exception`, we should be seeing the whole stacktrace anyway. :)